manager/state/raft/raft.go

```diff
@@ -992,10 +994,15 @@ func (n *Node) updateNodeBlocking(ctx context.Context, id uint64, addr string) e
 	if m == nil {
 		return errors.Errorf("member %x is not found for update", id)
 	}
+	if m.BackupAddrs == nil {
+		m.BackupAddrs = make(map[string]*types.Timestamp)
+	}
+	m.BackupAddrs[m.Addr] = ptypes.MustTimestampProto(time.Now())
 	node := api.RaftMember{
-		RaftID: m.RaftID,
-		NodeID: m.NodeID,
-		Addr:   addr,
+		RaftID:      m.RaftID,
+		NodeID:      m.NodeID,
+		Addr:        addr,
+		BackupAddrs: m.BackupAddrs,
```

Comment:
There probably should be Copy. Otherwise, it's easy to use a pointer to `Member` in a wrong way.
I'll do Copy in those places, though, because not changing `membership` is easier.